src/javascript/JContent/JContent.actions.jsx

```diff
@@ -204,14 +214,14 @@ export const jContentActions = registry => {
     registry.add('action', 'paste', {
         buttonIcon: <Paste/>,
         buttonLabel: 'jcontent:label.contentManager.contentPreview.paste',
-        targets: ['headerPrimaryActions:10', 'contentActions:3.91', 'narrowHeaderMenu:4'],
+        targets: ['headerPrimaryActions:10', 'contentActions:3.91', 'rootContentActions:3.91', 'narrowHeaderMenu:4'],
         component: PasteActionComponent
     });
     registry.add('action', 'pasteReference', {
         buttonIcon: <Paste/>,
         buttonLabel: 'jcontent:label.contentManager.contentPreview.pasteReference',
         referenceTypes: ['jnt:contentReference'],
-        targets: ['headerPrimaryActions:10.1', 'contentActions:3.92'],
+        targets: ['headerPrimaryActions:10.1', 'contentActions:3.92', 'rootContentActions:3.92'],
```

Comment:
I thought we don't want pasteAsReference in this menu?

Reply:
yes, you're right. I've removed this one.